admin/section/class-convertkit-settings-general.php

```diff
@@ -544,19 +556,68 @@ public function default_form_position_callback( $args ) {
 			$args['post_type'] . '_form_position',
 			esc_attr( $this->settings->get_default_form_position( $args['post_type'] ) ),
 			array(
-				'before_content'       => esc_html__( 'Before content', 'convertkit' ),
-				'after_content'        => esc_html__( 'After content', 'convertkit' ),
-				'before_after_content' => esc_html__( 'Before and after content', 'convertkit' ),
+				'before_content'       => sprintf(
```

Comment:
Changes e.g. `Before content` --> `Before page content` / `Before post content`, to make it clearer it'll position the form before/after/both of the entire Page / Post content, now that we have the `After element` option.